kubernetes/structure_persistent_volume_spec.go

```diff
@@ -27,6 +27,7 @@ func flattenAzureDiskVolumeSource(in *v1.AzureDiskVolumeSource) []interface{} {
 	att := make(map[string]interface{})
 	att["disk_name"] = in.DiskName
 	att["data_disk_uri"] = in.DataDiskURI
+	att["kind"] = string(*in.Kind)
```

Comment:
For pointer values we usually do a nil check and only pass it through when not nil.
Please see FSType below for a reference.

Reply:
Thanks @alexsomesan! I have made the change to check for ni and will add tests in the next couple of days and resubmit.